src/cascadia/PublicTerminalCore/HwndTerminal.cpp

```diff
@@ -230,7 +230,7 @@ HRESULT HwndTerminal::Initialize()
     RECT windowRect;
     GetWindowRect(_hwnd.get(), &windowRect);
 
-    const COORD windowSize{ gsl::narrow<short>(windowRect.right - windowRect.left), gsl::narrow<short>(windowRect.bottom - windowRect.top) };
+    const til::size windowSize{ windowRect.right - windowRect.left, windowRect.bottom - windowRect.top };

```

Comment:
next stop: `til::size<Unit>` where `Unit` is `til::unit::cell` and `til::unit::pixel` so we can _never_ accidentally convert a pixel size into a cell size